pygeoapi/templates/base.html

```diff
@@ -21,6 +21,21 @@
     {% endfor %}
     {% block extrahead %}
     {% endblock %}
+    <script>
```

Comment:
What do we want to accomplish here?

Reply:
The intention of this is to add a request for `<currentpage>?f=jsonld` and thereby injecting the JSON-LD representation (if present) of any resource that also has an HTML representation, as a datablock within the HTML. This provides SEO benefits. This could be done as part of the render itself, but this way means that the page loads and then makes a separate request just for JSON-LD.

Reply:
@alpha-beta-soup thanks for the info.  So this means an extra HTTP request on every page load?  Would rendering the result be an option (less HTTP requests, but work on rendering nonetheless).

Reply:
Yes, one additional request on page load, but it should not delay the page load as far as a human can tell (i.e. consuming HTML). We could render the result in the initial response, but that would delay the initial page load. My preference is the former, which is yours?

Reply:
The context request will probably be cached on the browser (or any client), maybe we should verify it has proper headers to be cached, maybe it helps to place the loading script at the footer of the page, any guidance on that available?

Reply:
Sorry, @pvgenuchten "the context request"? The JSON-LD included here is [not expanded](https://json-ld.org/spec/latest/json-ld-api/#expansion) and the `@context` is not requested separately. The one "additional request" I mention here is a request to `http://<currentpage>?f=jsonld` on top of the request for HTML at `http://<currentpage>`. The headers included for any `f=jsonld` resource are identical to the `f=json` resource, except for the MIME type (`application/ld+json` not `application/json`).

Where this JSON-LD data block is placed in the page is not critical. However the general consensus is the `head`, and that seems semantically appropriate to me, since the `head` includes information about the content of the page, which is exactly what we're doing with the JSON-LD representations.

Reply:
Sure the context should be placed in head, but the script that loads the context could be placed at the bottom of the page, eg

```
<html>
<head></head>
<body>
<main/>
<script that loads context and injects in head> 
</body>
</html>
```

Reply:
Oh, sure. That's possible, and  generally a good idea. I'll make that change.